react-components/package.json

```diff
@@ -24,6 +24,7 @@
     "test:coverage": "vitest run --dom --coverage",
     "test:playwright": "playwright test",
     "test:watch": "vitest watch --dom",
+    "check-circular-deps": "madge --circular --extensions ts,tsx --exclude 'setClippingPlanes.ts$|CommandButtons.tsx$' ./src",
```

Comment:
Why do we exclude these two files?

Reply:
The reason is that these two files still contain circular dependencies. We have another PR working on resolving those. But I want to add this check now, as I want to protect against any new circular dependencies getting added in the mean time, which happend a couple days ago...

Reply:
Sounds good! Just wanted to understand 👍🏻